src/bindist/autoemu.bat

```diff
@@ -1,8 +1,10 @@
 @echo off
 rem generic autoexec.bat for DOSEMU + any DOS
 rem must be edited in most circumstances
-path c:\dos;c:\windows\command;e:\dosemu
-lredir d: linux\fs\${DOSEMU_LIB_DIR}/freedos ro
+set PATH=c:\windows\command;e:\dosemu

```

Comment:
Your installer knows what did it install and
where. You should pass the needed parts
of the PATH via variables. For example if
you installed to D:\DOS, you can pass the
var PATH_XTRA that will contain D:\DOS
and append it to PATH.